tests/test_driver_alchemy_crud.py

```diff
@@ -17,78 +17,76 @@
 
 # TODO check if pytest has utilities for meta-programming of tests
 
+POSTGRES_CONNECTION = "postgresql://postgres:postgres@localhost:5432/indexd_tests"  # pragma: allowlist secret
 
-@util.removes("index.sq3")
-def test_driver_init_does_not_create_records():
+
+def test_driver_init_does_not_create_records(
+    combined_default_and_single_table_settings,
+):
     """
     Tests for creation of records after driver init.
     Tests driver init does not have unexpected side-effects.
     """
-    with sqlite3.connect("index.sq3") as conn:
-        driver = SQLAlchemyIndexDriver("sqlite:///index.sq3")
 
-        count = conn.execute(
-            """
-            SELECT COUNT(*) FROM index_record
-        """
-        ).fetchone()[0]
+    engine = create_engine(POSTGRES_CONNECTION)
+    driver = SQLAlchemyIndexDriver(POSTGRES_CONNECTION)
```

Comment:
i'm confused why you write `driver = SQLAlchemyIndexDriver(POSTGRES_CONNECTION)` in a bunch of tests that use `combined_default_and_single_table_settings`. When we use the single table it shouldn't be that driver(?)

there are 2 `test_driver_init_does_not_create_records` tests, the other one doesn't include `driver = SQLAlchemyIndexDriver(POSTGRES_CONNECTION)`, is it needed here?

Reply:
Went ahead and cleaned this up. I removed the `combined_default_and_single_table_settings` since these are tests that test direct queries to the old tables